server/service/integration_core_test.go

```diff
@@ -7391,7 +7391,6 @@ func (s *integrationTestSuite) TestOSVersions() {
 	// get OS versions
 	osv, err := s.ds.ListOperatingSystems(context.Background())
 	require.NoError(t, err)
-	require.Len(t, osv, 6) // includes fooOS from another test
```

Comment:
Removing to enable this test to be run independently of the suite (previous test adds an OS).  No value on the scope of the test method.